pkg/apis/core/types.go

```diff
@@ -3970,6 +3978,32 @@ type PodSchedulingGate struct {
 	Name string
 }
 
+// WorkloadReference identifies the Workload object and PodGroup membership
+// that a Pod belongs to. The scheduler uses this information to apply
+// workload-aware scheduling semantics.
+type WorkloadReference struct {
+	// Name defines the name of the Workload object this pod belongs to.
+	// Workload must be in the same namespace as the pod.
+	// It must be a DNS subdomain.
+	//
+	// +required
+	Name string
+
+	// PodGroup is the name of the PodGroup within the Workload that this pod
+	// belongs to. It must be a DNS label.
+	//
+	// +required
```

Comment:
one of the examples in KEP readme have workload without pod group defined. Is required here intentional?